Plots/Contours/NCL_eof_1_1.py

```diff
@@ -0,0 +1,67 @@
+"""
+NCL_eof_1_1.py
+===============
+Calculate EOFs of the Sea Level Pressure over the North Atlantic.
+
+Concepts illustrated:
+  - Calculating EOFs
+  - Drawing a time series plot
+  - Using coordinate subscripting to read a specified geographical region
+  - Rearranging longitude data to span -180 to 180
+  - Calculating symmetric contour intervals
+  - Drawing filled bars above and below a given reference line
+  - Drawing subtitles at the top of a plot
+  - Reordering an array
+
+Reproduces the NCL script found here:
+https://www.ncl.ucar.edu/Applications/Scripts/eof_1.ncl
+"""
+
+###############################################################################
+# Import the necessary python libraries
+import numpy as np
+import xarray as xr
+import cartopy
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.viz as gcv
+
+###############################################################################
+# Open the file for reading and print a content summary.
+ds = xr.open_dataset('../../data/netcdf_files/slp.mon.mean.nc')
+
+print(ds)
+
+
+###############################################################################
+# Flip longitude coordinates.
+print(f'Before flip, longitude range is [{ds["lon"].min().data}, {ds["lon"].max().data}].')
+
+ds["lon"] = ((ds["lon"] + 180) % 360) - 180
+
+# Sort longitudes, so that subset operations end up being simpler.
+ds["lon"] = np.sort(ds["lon"])
```

Comment:
This can't be right. I think you want `ds = ds.sortby("lon")`

Reply:
@dcherian Thanks for the advice.  But what I have seems to work; it produces the desired subset.   What is the reason to prefer one version over the other?   I ask as a naive user.  Thanks.

Reply:
you're changing the order of the coordinate values (`lat`) without changing the order of the associated variables such as `slp`.

Reply:
I see; I haven't plotted the subset values yet, so I did not have a way of knowing this.   Thank you.

Reply:
:+1: